app.js

```diff
@@ -1,15 +1,24 @@
 const express = require('express');
 const app = express();
 const { getTopics, getEndpoints } = require("./controllers/topics.controller.js");
+const { getArticleById } = require("./controllers/articles.controller.js");
 
 app.get('/api', getEndpoints)
 
 app.get('/api/topics', getTopics);
 
+app.get('/api/articles/:article_id', getArticleById)
+
 
 
 app.all('/*', (req, res, next) => {
     res.status(404).send({msg: 'path not found'})
 })
 
+app.use((err, req, res, next) => {
```

Comment:
You will need to add another error handler that examines the 'code' that comes back. Try looking at the console log from psql when you run your 'non number' test for this endpoint